Source/Blazorise/Components/Typography/UnorderedList.razor.cs

```diff
@@ -14,6 +14,8 @@ public partial class UnorderedList : BaseTypographyComponent
 
```

Comment:
Both lists implemention are very very similar and could almost have the same base, **ListBase** Not sure if worth it.

Reply:
nah